client/src/components/ScorecardSummaryPanel.vue

```diff
@@ -40,12 +42,47 @@ export default defineComponent({
   components: {
     ScorecardSummaryRow,
   },
+  setup() {
+    // Listen to state updates.
+    const geoState = useSelector((state) => state.geos) as GeoState;
+    const demographicData = useSelector((state) => state.demographicData) as DemographicDataState;
```

Comment:
nit/optional: the name of this feels a bit inconsistent with the geoState because of the Data portion. The state holds demographic data, same as geo state holds geographic data. It might be more appropriate to use 'data' in all state names or none.

Reply:
yeah good point. Renamed